go/pkg/framework/server.go

```diff
@@ -18,13 +18,18 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"runtime/debug"
 	"time"
 
 	"github.com/mlrun/mlrun/proto/build/health"
 
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
```

Comment:
```suggestion
	grpcmiddleware "github.com/grpc-ecosystem/go-grpc-middleware"
	grpcrecovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
```

[go package name conventiontions](https://go.dev/blog/package-names)